### PR TITLE
docs(user-tenant-service): update the readme

### LINKED ISSUE #878: Readme correction in user-tenant-service
Read me correction in user-tenant-service

### PR DESCRIPTION
**Pull request by sumiter92**
## Description

update the readme with correct values.

Fixes #878 

## Type of change

- [ ] Readme update

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] Code conforms with the style guide

### CODE REVIEW

File: services/user-tenant-service/README.md [hunk 1]
```diff
@@ -8,8 +8,7 @@
 
 ![npm (prod) dependency version (scoped)](https://img.shields.io/npm/dependency-version/@sourceloop/payment-service/@loopback/core)
 
-A Loopback Microservice primarily used for payment implementation to charge the payments for
-any client application.
+A Loopback Microservice primarily used for user-tenant implementation along with authenticcation service to set the api's for user-tenants.
```
Comment (akshatdubeysf) [inline services/user-tenant-service/README.md:11]: spelling mistake

File: services/user-tenant-service/README.md [hunk 2]
```diff
@@ -8,8 +8,7 @@
...
 
 ## Prerequiste
```
Comment (akshatdubeysf) [inline services/user-tenant-service/README.md:14]: Change this as well

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

File: services/user-tenant-service/README.md (hunk 1 above)
Comment (sumiter92) [reply services/user-tenant-service/README.md]: done

File: services/user-tenant-service/README.md (hunk 2 above)
Comment (sumiter92) [reply services/user-tenant-service/README.md:14]: done

**Review by akshatdubeysf — APPROVED** [review]

**Pull request closed without merging**